gradle.properties

```diff
@@ -0,0 +1,15 @@
+kotlin.code.style=official
```

Comment:
@dsvoronin review this. Maybe this file isn't needed

Reply:
i don't know for sure.
Seems like it's the same problem that was with gradle/wrapper.
Using `./gradlew -p <some project>` starts build within this directory and only reads it's own `gradle.properties` and wrapper settings.

So i don't know how to tell gradle to always use same settings for all included builds inside this repository. 
Probably via symlinks?

Also we can't control IDE run configurations, it is always runs with single project context (e.g. `-p <project>` you run any tasks within)

Reply:
[Gradle slack discussion](https://gradle-community.slack.com/archives/CAHSN3LDN/p1628684164052900)

Reply:
We can also generate this files for every included build with task like 
https://github.com/avito-tech/avito-android/blob/1f104034705e99206f68f2fe1fe6a5c0bd9263a0/build.gradle.kts#L8

it could be non invasive generation: only rewrite properties that we want to be the same, and left others in place, because some included builds may want to set it's own

Reply:
It could be done separately

Reply:
I think we need a research task to try and make the decision

Reply:
https://github.com/avito-tech/avito-android/pull/1168
sorry for rushing, but i just sit to play around this idea and seems like solve it

Please take a look